app/src/main/java/androidx/biometric/DeviceCredentialHandlerActivity.java

```diff
@@ -89,6 +89,7 @@ protected void onPause() {
         super.onPause();
 
         // Prevent the client from resetting the bridge in onPause if just changing configuration.
+        // JUST configuration change. lol
         final DeviceCredentialHandlerBridge bridge =
                 DeviceCredentialHandlerBridge.getInstanceIfNotNull();
         if (isChangingConfigurations() && bridge != null) {
```

Comment:
Yeah, so on rotation the `reset` in `DeviceCredentialHandlerBridge` I mentioned before is ignored and the activity is leaked